### PR TITLE
Added Unit Wisp Geometrical Model

### PR DESCRIPTION
**Pull request by dchang10**
Added model for a Unit Wisp. I hope the area scaling is correct

### CODE REVIEW

File: src/models/geometric_models.jl [hunk 1]
```diff
@@ -241,22 +241,26 @@ end
 A delta parabolic wisp in the image domain.
 The wisp is a parabolic line segment centered at zero, with roots ±1 and a yintercept of 1.
 """
-struct Wisp{F} <: GeometricModel end
-Wisp() = Wisp{Float64}()
-function Wisp(a, h) end
+struct UnitWisp{F} <: GeometricModel end
+UnitWisp() = UnitWisp{Float64}()
+function UnitWisp(a, h) end
```
Comment (ptiede) [inline src/models/geometric_models.jl:246]: for the UnitWisp(a,h) method we have two options. We can either delete the method or build a convenience function for a stretched wisp, something like

```julia
UnitWisp(a,h) = stretched(UnitWisp(), a, h)
``` 

although at that point we should probably call it Wisp right?

File: src/models/geometric_models.jl [hunk 2]
```diff
@@ -241,22 +241,26 @@ end
...
 radialextent(m::Wisp) = 1
```
Comment (ptiede) [inline src/models/geometric_models.jl:247]: This should be changed to 

```julia
@inline radialextent(::UnitWisp{T}) where {T} = one(T)
```

**Comment by ptiede** [comment]
I took a look at the implementation today and I think there is a small bug somewhere. The visibility should be equal to the intensity at the origin however I am not getting a unit flux. Instead, I get something ~1.35. I am guessing there is a small missing factor.

**Comment by dchang10** [comment]
For the wisp or the parabolic segment? Or both?

**Comment by ptiede** [comment]
> For the wisp or the parabolic segment? Or both?

This is for both. I think the length should drop out here for the Fourier transform so the function should be

```julia
function visibility_point(::ParabolicSegment{T}, u, v) where {T}
    ϵ = eps(T)
    vϵ = v + ϵ + 0im
    phase = exp(1im*π*(3/4 + 2*vϵ + u^2/(2vϵ)))
    #length = (√5 + asinh(2)/2)
    Δ1 = erf(√(π/(2vϵ))*exp(π*im/4)*(u-2vϵ))
    Δ2 = erf(√(π/(2vϵ))*exp(π*im/4)*(u+2vϵ))
    println(Δ1-Δ2)
    return phase/(√(2vϵ))*(Δ1-Δ2)/4
end
```
This also fixes the confusion about the normalization. There is no sinh issue so the ParabolicSegment and the Wisp should be equal.

**Comment by dchang10** [comment]
>I think the length should drop out here for the Fourier transform so the function should be
image

I checked the Fourier Transform. Apparently the normalization factor is 1/(2 a) and not 1/length. Seems a bit strange that it doesn't depend on the stretched height of the wisp

**Comment by dchang10** [comment]
> I think the length should drop out here for the Fourier transform so the function should be

image

Yep, your totally right. The normalization factor is actually proportional to "a" and not the length. I made the fallacious assumption that the integral of a delta of a function that describes a level curve is the area of that curve. I just checked it for this wisp and it in fact just "a"

**Comment by ptiede** [comment]
Alright, I did a rebase off main and fixed the normalization. Given that the Parabolic segment should equal the Wisp I decided to just ax it. Let me know if you object to it!

**Comment by ptiede** [comment]
Merged! Great job Dominic.

**Pull request merged**